src/lib/annotations/AnnotationDialog.js

```diff
@@ -163,9 +166,7 @@ class AnnotationDialog extends EventEmitter {
         this.unbindDOMListeners();
 
         // Cancel any unsaved annotations
-        if (!this.hasAnnotations) {
```

Comment:
Double check that this is ok to move on all other annotation types. I feel like there was a reason I added that check just forgot to add a comment explaining why